src/styles/main.css

```diff
@@ -0,0 +1,84 @@
+body {
+  margin: 0;
+}
+
+.header {
+  margin: 20px auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  display: -webkit-box;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
```

Comment:
Why do you need these properties?
You can just use default display: flex and other flexbox properties
https://css-tricks.com/snippets/css/a-guide-to-flexbox/